client/galaxy/scripts/mvc/library/library-foldertoolbar-view.js

```diff
@@ -1828,13 +1828,13 @@ var FolderToolbarView = Backbone.View.extend({
                 "<h4>Which datasets?</h4>",
                 '<form class="form-inline">',
                 '<label class="radio-inline">',
-                '<input type="radio" name="radio_elements" id="selection_radio" value="selection" <% if (!selected_datasets) { %> disabled <% } %> > current selection',
+                '<input type="radio" name="radio_elements" id="selection_radio" value="selection" <% if (!selected_datasets) { %> disabled <% } else { %> checked <% } %> > current selection',
```

Comment:
very useful, I wanted to add a validator for the radio selection but this is better